lib/features/context-pad/ContextPadProvider.js

```diff
@@ -26,6 +26,8 @@ import {
   hasPrimaryModifier
 } from 'diagram-js/lib/util/Mouse';
 
+import ICONS from './ContextPadIcons';
```

Comment:
Feels kinda odd to now spread "icons" across the whole code base.

What about collecting all icons in a shared place? They can be bundled once, as needed, and consumed from anywhere.

I could imagine to provide icons through `lib/icons` to start with.